app/pages/project/affinity/AntiAffinityGroupPage.tsx

```diff
@@ -152,9 +156,35 @@ export default function AntiAffinityPage() {
     getCoreRowModel: getCoreRowModel(),
   })
 
+  const disabledReason = () => {
+    // https://github.com/oxidecomputer/omicron/blob/main/nexus/db-queries/src/db/datastore/affinity.rs#L66
```

Comment:
link to a commit (press `y` to get a canonical URL) so the line number is guaranteed to stay correct